org-drill-table.el

```diff
@@ -154,7 +155,11 @@ and the row value."
       (if (zerop idx) (org-insert-subheading nil) (org-insert-heading))
       (insert header)
       (newline)
-      (insert value))))
+      (org-indent-line)
+      (insert value)
+      (org-indent-line)
```

Comment:
Rather than having parens hanging like in C languages, closing parens should be joined to the same line. E.g., prefer:

```lisp
(foo 
  (bar
    (baz)))
```

to hanging style:

```lisp
(foo
  (bar
    (baz)
  )
)
```